packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,294 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/97972/
+  - title: "Migrate 'ThemeData.toggleableActiveColor' to individual themes"
+    date: 2022-05-18
+    element:
+      uris: [ 'material.dart' ]
+      constructor: ''
+      inClass: 'ThemeData'
+    changes:
+    - kind: 'removeParameter'
+      name: 'toggleableActiveColor'
+    - kind: 'addParameter'
+      index: 96
```

Comment:
Why is this `checkboxTheme` 96, and the one below `97`? Shouldn't they be the same since they represent the same parameter? I would check all of these indices. Since this adds a lot of code, we should make sure it is getting inserted in the right place.

Reply:
This is `checkboxTheme()` and below is the `checkboxTheme.copyWith`. If I add matching indices for both, dart fix throws `Error from the analysis server: Exception while getting bulk fixes:`.